### PR TITLE
feat: replace mjpg-streamer with crowsnest

### PR DESCRIPTION
**Pull request by KwadFan**
Signed-off-by: Stephan Wendel <me@stephanwe.de>

### CODE REVIEW

File: scripts/crowsnest.sh [hunk 1]
```diff
@@ -0,0 +1,229 @@
...
+#         status_msg "Installing required packages..."
+#         if ! sudo apt-get install --yes "${install[@]}"; then
+#             log_error "failure while installing required crowsnest basic packages"
+#             error_msg "Installing required packages failed!"
+#             exit 1
+#         fi
+#     else
+#         ok_msg "All basic dependencies met, nothing to do!"
+#     fi
+# }
```
Comment (pedrolamas) [inline scripts/crowsnest.sh:111]: Any specific reason as to why leave this here commented? Might as well just remove it, no?

Comment (KwadFan) [reply scripts/crowsnest.sh:111]: Well, obviously I was lost in my own code :rofl: 

No, will remove that asap

Comment (pedrolamas) [reply scripts/crowsnest.sh:111]: Nevertheless, I tried your PR and it "worked fine on my machine"! 😁

**Comment by KwadFan** [comment]
Requested your Review @pedrolamas

**Review by pedrolamas — APPROVED** [review]
As far as I can see, this LGTM!

**Pull request merged**